### DISCUSSION TITLE
Can't see my data in a my scree

### DISCUSSION THREAD

**Discussion by drummingman77** [Budibase/budibase#9017] [Help]
I have a new BudiBase DB that has two rows and a half dozen columns. When I go to the "Design" tab and add a Data Provider element, the source shows my DB name "Properties" but none of the DB entries I have show up. I set permissions to public, too. What am I missing?

**Comment by andz-bb** [comment]
hey @drummingman77, could you send me an app export and I can have a look? You can email it to support@budibase.com or send it to me directly on [our Discord server here.](https://discord.gg/E3YXNAg4m5)
